SU2_CFD/src/output/CFVMOutput.cpp

```diff
@@ -0,0 +1,85 @@
+/*!
+ * \file CFVMOutput.cpp
+ * \brief Main subroutines for Finite Volume Method output
+ * \author T. Kattmann
+ * \version 7.1.1 "Blackbird"
+ *
+ * SU2 Project Website: https://su2code.github.io
+ *
+ * The SU2 Project is maintained by the SU2 Foundation
+ * (http://su2foundation.org)
+ *
+ * Copyright 2012-2021, SU2 Contributors (cf. AUTHORS.md)
+ *
+ * SU2 is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * SU2 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with SU2. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "../../include/output/CFVMOutput.hpp"
+#include "../../../Common/include/geometry/CGeometry.hpp"
+
+
+CFVMOutput::CFVMOutput(CConfig *config, unsigned short nDim, bool fem_output) : COutput (config, nDim, fem_output){ }
+
+CFVMOutput::~CFVMOutput(void){ }
+
+void CFVMOutput::AddCommonFVMOutputs(const CConfig *config) {
+
+  // Grid coordinates
+  AddVolumeOutput("COORD-X", "x", "COORDINATES", "x-component of the coordinate vector");
+  AddVolumeOutput("COORD-Y", "y", "COORDINATES", "y-component of the coordinate vector");
+  if (nDim == 3)
+    AddVolumeOutput("COORD-Z", "z", "COORDINATES","z-component of the coordinate vector");
+
+  // Mesh quality metrics
+  AddVolumeOutput("ORTHOGONALITY", "Orthogonality", "MESH_QUALITY", "Orthogonality Angle (deg.)");
+  AddVolumeOutput("ASPECT_RATIO",  "Aspect_Ratio",  "MESH_QUALITY", "CV Face Area Aspect Ratio");
+  AddVolumeOutput("VOLUME_RATIO",  "Volume_Ratio",  "MESH_QUALITY", "CV Sub-Volume Ratio");
+
+  AddVolumeOutput("RANK", "rank", "MPI", "Rank of the MPI-partition");
```

Comment:
Have you tested this? i.e. write a restart and actually use it?

Reply:
f**k. well yes ... i didn't think of that. it writes the fields for mesh quality directly after coord  .. which means I can revert the Coord stuff as I would rather not like to mess with the skip_var things.

Reply:
Good thing I dont trust myself and put this in its own commit to make it easily revertable

Reply:
You can always make another function called AddCoordinates, it does not have to be all or nothing

Reply:
true. It is up to others to deal in absolutes :) added that